### ISSUE TITLE
How LNURL can work with this standard?

### ISSUE THREAD

**Issue by AugustoResende** [sbddesign/bip21-site#70]
Related:

https://github.com/fiatjaf/lnurl-rfc/issues/177

**Comment by tom-playasia** [comment]
Yes I also wonder. The QR code would be much smaller if we can just create a payrequest.

**Comment by sbddesign** [comment]
As I mentioned [here in the LUDS conversation](https://github.com/lnurl/luds/issues/177#issuecomment-1357663455), there is nothing to stop anyone who wants to add an LNURL parameter to a BIP21. BIP21 is already flexible enough to support this. Getting wallets to be compatible with this is another discussion entirely.

I'm going to close this issue since it's a little outside the scope of this website's aims.
